pkg/virt-launcher/virtwrap/converter/converter_test.go

```diff
@@ -2322,6 +2322,62 @@ var _ = Describe("Converter", func() {
 			// Should still apply hypervisor adjustments
 			Expect(domain.Spec.Type).To(Equal("hyperv"))
 		})
+
```

Comment:
In this PR we're adding logging. We're not changing selection or fallback logic at all. So if we add any tests, we should check logging (if it's even possible) but not the selection logic.